integration/combination/test_api_settings.py

```diff
@@ -0,0 +1,180 @@
+from io import BytesIO
+
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
+
+import requests
+from parameterized import parameterized
+
+from integration.helpers.base_test import BaseTest
+
+from PIL import Image
+
+
+class TestApiSettings(BaseTest):
+    def test_method_settings(self):
+        self.create_and_verify_stack("combination/api_with_method_settings")
+
+        rest_api_id = self.get_physical_id_by_type("AWS::ApiGateway::RestApi")
+        apigw_client = self.client_provider.api_client
+        response = apigw_client.get_stage(restApiId=rest_api_id, stageName="Prod")
+
+        wildcard_path = "*/*"
+
+        method_settings = response["methodSettings"]
+        self.assertTrue(wildcard_path in method_settings, "MethodSettings for the wildcard path must be present")
+
+        wildcard_path_setting = method_settings[wildcard_path]
+        print(wildcard_path_setting)
```

Comment:
If this is necessary, can you use another way like `LOG.info`?

Reply:
Maybe a leftover from debugging

Reply:
deleted, just a leftover from debugging